src/config/backpack/base.php

```diff
@@ -272,9 +272,23 @@
     // Username column for authentication
     // The Backpack default is the same as the Laravel default (email)
     // If you need to switch to username, you also need to create that column in your db
-    'authentication_column'      => 'email',
+    'authentication_column' => 'email',
     'authentication_column_name' => 'Email',
 
+    // As default backpack will use email column as username
+    // If we change email column as "authentication_column", then we can set email column here to not break recovery password and keep working route:list
+    // Be sure set email column as fillable in User model
+    /**
+     * If dont use email_column = email, then add to user Model:
+     * use CanResetPassword;.
+     *
+     * public function getEmailForPasswordReset()
+     * {
+     *   return $this->{backpack_email_column()};
+     * }
+     */
+    'email_column' => 'email',
```

Comment:
We might need this to make it non-breaking:

```suggestion
    // 'email_column' => 'email',
```